vmck/backends/qemu.py

```diff
@@ -47,11 +48,12 @@ def task_group(job, options):
     vm_port = random_port()
 
     image_artifact = {
-        'getterSource': settings.QEMU_IMAGE_URL,
+        'getterSource': urljoin(settings.QEMU_IMAGE_PATH_PREFIX,
```

Comment:
perfect

Reply:
We should make sure there's no `../` in the `image_path` provided by the user. Also, I think if they provide an absolute URL, it will ignore our prefix and just use the user's URL. This is a major security flaw :) We should assert that the final URL begins with `settings.QEMU_IMAGE_PATH_PREFIX`.